packages/web-client/tests/integration/components/workflow-thread-test.ts

```diff
@@ -26,46 +28,74 @@ module('Integration | Component | workflow-thread', function (hooks) {
+      message: 'Hello world',
+    });
+    let postable2 = new WorkflowMessage({
+      author: { name: 'cardbot' },
+      message: 'How are you today?',
+    });
+    let postable3 = new WorkflowMessage({
+      author: { name: 'joe' },
+      message: 'Fine thank you.',
+    });
+    let postable4 = new WorkflowMessage({
+      author: { name: 'steve' },
+      message: 'One more thing...',
+    });
+
     workflow.milestones = [
       new Milestone({
         title: 'First milestone',
-        postables: [
-          new WorkflowMessage({
-            author: { name: 'cardbot' },
-            message: 'Hello world',
-          }),
-        ],
+        postables: [postable1],
         completedDetail: 'You are number one!',
       }),
       new Milestone({
         title: 'Second milestone',
-        postables: [
-          new WorkflowMessage({
-            author: { name: 'cardbot' },
-            message: 'How are you today?',
-          }),
-          new WorkflowMessage({
-            author: { name: 'joe' },
-            message: 'Fine thank you.',
-          }),
-        ],
+        postables: [postable2, postable3],
         completedDetail: 'Had to do number two!',
       }),
     ];
-    workflow.milestones[0].postables[0].timestamp = new Date(
-      Date.parse('2021-05-20 00:00:00')
-    );
-    workflow.milestones[1].postables[0].timestamp = new Date(
-      Date.parse('2021-05-20 00:00:00')
-    );
-    workflow.milestones[1].postables[1].timestamp = new Date(
-      Date.parse('2021-05-21 00:00:00')
-    );
+    workflow.epilogue.postables = [postable4];
+    postable1.timestamp = new Date(Date.parse('2021-05-20 00:00:00'));
+    postable2.timestamp = new Date(Date.parse('2021-05-20 00:00:00'));
+    postable3.timestamp = new Date(Date.parse('2021-05-21 00:00:00'));
+    postable4.timestamp = new Date(Date.parse('2021-05-21 00:00:00'));
     this.set('workflow', workflow);
     workflow.attachWorkflow();
     await render(hbs`
       <WorkflowThread @workflow={{this.workflow}} />
     `);
     assert.dom('[data-test-date-divider]').exists({ count: 2 });
   });
+
+  test('it renders epilogue posts after the workflow is complete', async function (assert) {
+    let workflow = new ConcreteWorkflow(this.owner);
+    let postable1 = new WorkflowPostable({ name: 'cardbot' });
+    workflow.milestones = [
+      new Milestone({
+        title: 'First milestone',
+        postables: [postable1],
+        completedDetail: 'You are number one!',
+      }),
+    ];
+    workflow.epilogue = new PostableCollection([
+      new WorkflowMessage({
+        author: { name: 'cardbot' },
+        message: 'This is the epilogue',
+      }),
+    ]);
+    this.set('workflow', workflow);
+    workflow.attachWorkflow();
+    await render(hbs`
+      <WorkflowThread @workflow={{this.workflow}} />
+    `);
+    assert.dom('[data-test-epilogue][data-test-postable]').doesNotExist();
+    postable1.isComplete = true;
+    await settled();
```

Comment:
This is interesting. The tests fail if I remove this, but I can't see what asynchronous work is going on that would require settled - is it simply the DOM changes?

Reply:
@Aierie `settled` waits for any "test waiters" to finish, including any queued tasks on the Ember run loop. An alternative that might be clearer would be to wait for the DOM state that we expect.

Reply:
I see, thanks. So these waiting items might block ember from rerendering?

Reply:
No, the `settled` test waiter "waits" for Ember to complete rendering (among other things). There are a bunch of layers here, but you can have a look at https://github.com/emberjs/ember-test-helpers/blob/master/addon-test-support/%40ember/test-helpers/settled.ts as a starting point.

That being said, this conversation has convinced me I should use an approach that is more intention-revealing, so I'm going to make that change.